gateway/cli/src/main.rs

```diff
@@ -324,7 +350,9 @@ async fn main() -> anyhow::Result<()> {
                 block_height,
                 max_retries,
                 retry_delay_seconds,
+                exclude_config,
```

Comment:
You're just throwing away the config below anyway since the only field that is used is `block_height`, so this doesn't need to be a parameter. Just set it to true in the actual `get_info` RPC